reference/api/documents.mdx

```diff
@@ -4,31 +4,105 @@ The `/documents` route allows you to create, manage, and delete documents.
 
 [Learn more about documents.](/learn/core_concepts/documents)
 
-## Get documents
+## Get documents with POST
```

Comment:
Following the same logic as with the search references, to not confuse the user we might just call it `Get Documents` and not add POST.

Reply:
There are two endpoints: 
- Get documents with GET
- Get documents with POST

Since both allow users to do the same thing(almost): get all documents or get some documents based on a filter, we decided to go with the same heading names: Get documents with GET/POST.

Reply:
In the search references, there is `search` and `search with GET`.

Since we want to promote Get Documents with POST as the default value, adding `POST` may be confusing for a user that does not understand HTTP methods.

Reply:
>In the search references, there is search and search with GET.

It's search with GET and search with POST, same as here.

>Since we want to promote get Documents with POST as the default value, adding POST may be confusing for a user that does not understand HTTP methods.

About promoting POST: I added it to the top and added a warning under the GET route, saying it will be deprecated in the future so use the POST rooute

Not knowing about HTTP verbs: I'm assuming users would have some idea. If they don't, we have the warning that says we recommend using POST instead of GET

Reply:
My bad, I did not see the search with POST one!